src/containers/MapControls.jsx

```diff
@@ -268,11 +276,11 @@ class SelectorPanel extends Component {
   //Download the current results into a CSV.
   prepareCsv(e) {
     //First things first: make sure the user sees what she/he is going to download.
-    this.updateMe(null);
+    //this.updateMe(null);  //ERROR: TODO: https://github.com/zooniverse/wildcam-gorongosa-education/issues/229
 
     this.setState({
```

Comment:
This seems like a holdover from the old React thinking; this qualifies for an across-the-board upgrade.